COLLABORATOR_GUIDE.md

```diff
@@ -24,8 +24,9 @@ understand the project governance model as outlined in
 
 ## Issues and Pull Requests
 
-Courtesy should always be shown to individuals submitting issues and
-pull requests to the Node.js project.
+Courtesy should **always** be shown to individuals submitting issues and pull
```

Comment:
> Not an objection, but just a comment that I try to be very sparing with bold, italics, etc. It gets out of hand pretty easily.

Ack. I felt this is a point worth emphasising both to Collaborators, and to contributors who read this (as on most screens it appears above the fold)